Source/WebCore/platform/mediastream/mac/BaseAudioSharedUnit.cpp

```diff
@@ -44,6 +44,8 @@ BaseAudioSharedUnit::BaseAudioSharedUnit()
 
 void BaseAudioSharedUnit::addClient(CoreAudioCaptureSource& client)
 {
+    RELEASE_LOG_ERROR(WebRTC, "BaseAudioSharedUnit::addClient %p", this);
```

Comment:
Ditto

Reply:
This should be removed, I did it in the new patch